android/app/build.gradle

```diff
@@ -76,13 +76,22 @@ import com.sun.org.apache.xalan.internal.xsltc.compiler.Copy
  *   extraPackagerArgs: []
  * ]
  */
+
+/*
+ Enabling Hermes on x86 devices is crashing the app after a few reloads,
+ this flag can be used to disable Hermes while building app.
+ https://github.com/status-im/status-mobile/issues/14031
+ */
+
+def disableHermes = new Boolean(System.getenv('DISABLE_HERMES') ?: false)
```

Comment:
If you don't check the string value of `DISABLE_HERMES` variable, then setting `DISABLE_HERMES=false` will result in `disableHermes` being `true`.

Reply:
Hi, thank you for reviewing pr
As we discussed, `new Boolean()` only treats 'true'/true as truthy and all other args as falsey.

But Still, now replaced with
`def disableHermes = System.getenv('DISABLE_HERMES') == 'true'`

as it is more clear and self explanatory. Thank you